client/src/components/Search/Search.js

```diff
@@ -46,28 +81,82 @@ export default class Search extends React.Component {
+                                             tags={tags} materials={materials}/>
+                                    <SearchInfo count={count} page={page} limit={limit} sort={sort}
+                                                onLimitChange={onLimitChange}
+                                                onSortChange={onSortChange}
+                                                showDistanceSort={lat && lon}/>
+                                </div>
+                                <div className="search-results">
+                                    <SearchResults monuments={monuments} limit={limit} page={page}/>
+                                </div>
+                                <div className="pagination-container">
+                                    <Pagination count={pageCount}
+                                                page={page - 1}
+                                                onPage={page => this.handlePageChange(page + 1)}/>
+                                </div>
+                            </div>
+                            <div className="map-row">
+                                <MapResults monuments={monuments} zoom={lat && lon ? 10 : 4} center={lat && lon ? [lat, lon] : null}/>
+                            </div>
+                        </>
+                    }
+                    {
+                        ((this.state.orientation === 'portrait' && window.innerWidth <= 414) ||
+                        (this.state.orientation === 'landscape' && window.innerWidth <= 823 && window.innerHeight <= 414)) && <>
+                            <div className="search-full-page">
+                                <div className="search-header">
+                                    <Filters onChange={filters => onFilterChange(filters)}
+                                             showDistance={lat && lon} distance={distance}
+                                             tags={tags} materials={materials}/>
+                                    <SearchInfo count={count} page={page} limit={limit} sort={sort}
+                                                onLimitChange={onLimitChange}
+                                                onSortChange={onSortChange}
+                                                showDistanceSort={lat && lon}/>
+                                </div>
+                                <div className="search-results">
+                                    <SearchResults monuments={monuments} limit={limit} page={page}/>
+                                </div>
+                                <div className="pagination-container">
+                                    <Pagination count={pageCount}
+                                                page={page - 1}
+                                                onPage={page => this.handlePageChange(page + 1)}/>
+                                </div>
+                            </div>
+                        </>
+                    }
```

Comment:
Can't this be done via media breakpoints inside `.scss` files? This is a mess and makes it extremely hard to edit in the future.

Reply:
I don't think so. I could do it for the different class names, but I need to rerender new components or get rid of components at certain widths.

Reply:
But can't you have them `display: block` (or whatever display you want) by default then at different sizes have the `display: none` as you need?

Reply:
I never tried it that way, I really only knew how to do this way efficiently. I can try that way and see how it goes.

I still think it needs to be at least 2 things because I am changing where a component is being rendered, not just visibility, but maybe that can also be done in the scss file. I just don't know how to do that efficiently

Reply:
I can get it done that way, I just don't know if I can get it done that way by 5. I will try though